CMakeLists.txt

```diff
@@ -110,6 +109,7 @@ add_library(maliit-connection STATIC ${CONNECTION_SOURCES})
 target_link_libraries(maliit-connection Qt5::Core Qt5::DBus Qt5::Gui maliit-common)
 if(enable-wayland)
     target_link_libraries(maliit-connection Wayland::Client PkgConfig::XKBCOMMON)
+    target_include_directories(maliit-connection PRIVATE ${Qt5WaylandClient_PRIVATE_INCLUDE_DIRS})
```

Comment:
You should be able to use the Qt5::WaylandClientPrivate imported target instead

Reply:
This seems to require adding it to `target_link_libraries` which creates an unneeded link to `libQt5WaylandClient.so.5`, and I don't think we want that.

Reply:
We don't link against that already? Ok, fair enough then